### PR TITLE
Tweak archive

### PR DESCRIPTION
**Pull request by ericnewcomer**
- Use daily and monthly everywhere for api filters
 - Fix queries and templates to match model changes for archive ui

### CODE REVIEW

File: temba/archives/views.py [hunk 1]
```diff
@@ -34,8 +34,10 @@ def get_queryset(self, **kwargs):
             queryset = super().get_queryset(**kwargs)
 
             # filter by our archive type
-            return queryset.filter(archive_type=self.get_archive_type()).exclude(
-                period=Archive.PERIOD_DAILY, rollup_id__isnull=False
+            return (
+                queryset.filter(org=self.org, archive_type=self.get_archive_type())
+                .exclude(rollup_id__isnull=False)
+                .exclude(record_count=0)
```
Comment (nicpottier) [inline temba/archives/views.py]: man this is automatic? so cool!

Comment (ericnewcomer) [reply temba/archives/views.py]: what is? dodoing the return?

Comment (nicpottier) [reply temba/archives/views.py]: Ya

Comment (nicpottier) [reply temba/archives/views.py]: And ya, oops on that query.

Comment (ericnewcomer) [reply temba/archives/views.py]: i wish my name was a verb

Comment (nicpottier) [reply temba/archives/views.py]: well we could start using that when someone forgets to filter by org maybe? ;)

Comment (ericnewcomer) [reply temba/archives/views.py]: WFM: https://github.com/nyaruka/rapidpro/blame/master/temba/archives/views.py#L37

Comment (ericnewcomer) [reply temba/archives/views.py]: ...just don't crawl up that blame tree.

Comment (ericnewcomer) [reply temba/archives/views.py]: https://github.com/nyaruka/rapidpro/blame/7c4ab3364fb39e66d3b10680ca8031e400633791/temba/archives/views.py#L37

Comment (nicpottier) [reply temba/archives/views.py]: how far does the rabbit hole go!

**Pull request merged**